ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly.yaml

```diff
@@ -990,6 +990,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-ipsec-azurefile-csi
+- as: azure-ipi-sharednetwork-ovn-ipv4-subnet-amd-f28-destructive
+  cron: 25 23 27 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-azure-ipi-sharednetwork-ovn-ipv4-subnet
+- as: azure-ipi-sharednetwork-ovn-ipv4-subnet-arm-f14
+  cron: 14 14 6,22 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
```

Comment:
@jinyunma saw there is `OVN_IPV4_INTERNAL_SUBNET`, but not set the value in job,  so that mean it will be `101.64.0.0/16` by default,  is there any validation  to make sure ovn is setting correct after cluster is ready?

Reply:
It is configured in install provision chain [here](https://github.com/openshift/release/pull/59148/files#diff-e47e522329e63555c2458fad08f84ba4d59b7b6f19f6ab96e3be44f13ea8d295R13-R14).

We have post-check step [cucushift-installer-check-network-ovn-ipv4-subnet-commands.sh](https://github.com/openshift/release/pull/59148/files#diff-84eca61557e486b17f3abef7d04961c9959ccbe5678cd19d38b83a8192bd5e7a)  to make sure that setting is correct in network spec on running cluster.

Reply:
ok, thanks,  I should be missed that.  lgtm